src/models/ws-stream.js

```diff
@@ -31,13 +34,13 @@ function openWebSocket (url, authorization) {
     });
 
     ws.on('close', () => {
-      Logger.debug('Websocket closed.');
+      Logger.debug('WebSocket closed.');
       clearInterval(pingInterval);
       sink(new Bacon.End());
     });
 
     ws.on('error', () => {
-      Logger.debug('Websocket closed.');
+      Logger.debug('WebSocket closed.');
```

Comment:
Maybe log the error and put a different message to distinguish between normal closing and errors? Having the same messages made debugging more difficult

Reply:
Good catch.